resources/views/game-detail.blade.php

```diff
@@ -0,0 +1,129 @@
+@extends('layouts.app')
+
+@section('page-title')
+    {{ $game->title." || Game Store" }}
+@endsection
+
+@section('content')
+    <div class="games-details-area section pt-100 pt-lg-80 pt-md-70 pt-sm-60 pt-xs-50 pb-xs-45">
+        <div class="container">
+            @if (session('error'))
+            <div class="alert alert-danger">
+                <i class="fa fa-exclamation-triangle"></i> {{ session('error') }}
+            </div>
+            @endif
+            <div class="row">
+                <div class="col-lg-8">
+                    <div class="row">
+                        <div class="col-12">
+                            <div class="game-image-gallery-wrap">
+                                <div class="game-description mb-45">
+                                    <a href="{{ url()->previous() }}"><i class="fa fa-arrow-left"></i> {{ trans('text.game-detail.back') }}</a>
+                                </div>
+                                <div class="game-image-large">
+                                   <div class="game-image img-full">
+                                    <img src="{{ asset(config('link.games').$game->id.'/large/game-large1.jpg') }}" alt="">
```

Comment:
```suggestion
                                    <img src="{{ asset(config('link.games') . $game->id . '/large/game-large1.jpg') }}" alt="">
```
cho string vào config